locations/spiders/amctheatres.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import re
+import json
+
+from locations.items import GeojsonPointItem
+
+
+class AmcTheatresSpider(scrapy.Spider):
+    name = "amctheatres"
+    allowed_domains = ['amctheatres.com']
+    start_urls = (
+        'https://www.amctheatres.com/sitemaps/sitemap-theatres.xml',
+    )
+
+    def parse(self, response):
+        response.selector.remove_namespaces()
+        city_urls = response.xpath('//url/loc/text()').extract()
+        for path in city_urls:
+            yield scrapy.Request(
+                path.strip(),
+                callback=self.parse_store,
+            )
+
+    def parse_store(self, response):
+        if response.xpath('/html/body/div/div/main/section/div/div/div/div/div/div/h4/text()').extract_first() == "Error 500":
+            pass
+        else:
+            properties = {
+                'name': response.xpath('/html/body/div/div/main/div/section/div/div/div/div/div/div/h1/text()').extract_first(),
```

Comment:
Rather than parsing the parsing the HTML body you can probably use the `<meta>` tags like so:

```python
response.xpath('//meta[@name="amc:theatre-name"]/@content').extract_first()
```